apps/chat-e2e/src/assertions/sideBarEntityAssertion.ts

```diff
@@ -101,6 +129,41 @@ export class SideBarEntityAssertion<T extends SideBarEntities> {
       .toBe(expectedColor);
   }
 
+  public async assertEntityCheckboxColor(
+    entity: TreeEntity,
+    expectedColor: string,
+  ) {
+    const entityCheckboxColor = await this.sideBarEntities
+      .getEntityCheckbox(entity.name, entity.index)
+      .evaluate((el) => window.getComputedStyle(el).getPropertyValue('color'));
+    expect
+      .soft(entityCheckboxColor, ExpectedMessages.iconColorIsValid)
+      .toBe(expectedColor);
+  }
+
+  public async assertEntityCheckboxBorderColors(
+    entity: TreeEntity,
+    expectedColor: string,
+  ) {
+    const entityCheckboxBorderColors = await this.sideBarEntities
+      .getEntityCheckbox(entity.name, entity.index)
+      .evaluate((el) => {
```

Comment:
please, use already implemented in `BaseElement` method `getAllBorderColors()`

Reply:
done